deploy/crds/workspaces.ecd.eclipse.org_devworkspacetemplates_crd.yaml

```diff
@@ -0,0 +1,491 @@
+                          executed
+                        type: string
+                    required:
+                    - commandLine
+                    type: object
+                  type:
+                    description: Type of workspace command
+                    enum:
+                    - Exec
+                    - VscodeTask
+                    - VscodeLaunch
+                    - Custom
+                    type: string
+                  vscodeLaunch:
+                    description: VscodeLaunch command
+                    properties:
+                      alias:
+                        type: string
+                      attributes:
+                        additionalProperties:
+                          type: string
+                        type: object
+                      inlined:
+                        description: Embedded content of the vscode configuration
+                          file
+                        type: string
+                      locationType:
+                        description: Type of Vscode configuration command location
+                        type: string
+                      url:
+                        description: Location as an absolute of relative URL
+                        type: string
+                    type: object
+                  vscodeTask:
+                    description: VscodeTask command
+                    properties:
+                      alias:
+                        type: string
+                      attributes:
+                        additionalProperties:
+                          type: string
+                        type: object
+                      inlined:
+                        description: Embedded content of the vscode configuration
+                          file
+                        type: string
+                      locationType:
+                        description: Type of Vscode configuration command location
+                        type: string
+                      url:
+                        description: Location as an absolute of relative URL
+                        type: string
+                    type: object
+                type: object
+              type: array
+            components:
+              description: List of the workspace components, such as editor and plugins,
+                user-provided containers, or other types of components
+              items:
+                properties:
+                  cheEditor:
+                    description: CheEditor component
+                    properties:
+                      alias:
+                        type: string
+                      locationType:
+                        description: Type of plugin location
+                        enum:
+                        - RegistryEntry
+                        - Uri
+                        type: string
+                      memoryLimit:
```

Comment:
Does it make sense to include `computeLimit` as well? Similarly for requests?

Reply:
It might make sense yes, but in fact this is generated from the existing description that was already there in the `DevWorkspaceTemplateSpec` Api Go struct. The only new thing is that this struct is now used in 2 main Custom Resource APIs.
 
So this change is not really part of the source changes done for this PR.

Do you think we could postpone additional fixes in the overall GO API to a distinct issue ?

Reply:
:+1: